```
Running Sphinx v6.2.1
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/ceph/envs/66936/lib/python3.9/site-packages/sphinx/registry.py", line 442, in load_extension
    mod = import_module(extname)
  File "/home/docs/.asdf/installs/python/3.9.25/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/docs/checkouts/readthedocs.org/user_builds/ceph/envs/66936/lib/python3.9/site-packages/sphinxcontrib/seqdiag.py", line 19, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/ceph/envs/66936/lib/python3.9/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/home/docs/checkouts/readthedocs.org/user_builds/ceph/envs/66936/lib/python3.9/site-packages/sphinx/application.py", line 229, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/ceph/envs/66936/lib/python3.9/site-packages/sphinx/application.py", line 404, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/ceph/envs/66936/lib/python3.9/site-packages/sphinx/registry.py", line 445, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname,
sphinx.errors.ExtensionError: Could not import extension sphinxcontrib.seqdiag (exception: No module named 'pkg_resources')
Extension error:
Could not import extension sphinxcontrib.seqdiag (exception: No module named 'pkg_resources')
```

hmm.

--- a/src/common/obj_bencher.cc
+++ b/src/common/obj_bencher.cc
@@ -333,13 +333,14 @@ int ObjBencher::aio_bench(
     }
 
     data.start_time = mono_clock::now();
-    out(cout) << "Cleaning up (deleting benchmark objects)" << std::endl;
+    out(cerr) << "Cleaning up (deleting benchmark objects)" << std::endl;
 
     r = clean_up(num_objects, prev_pid, concurrentios);
     if (r != 0) goto out;
 
     timePassed = mono_clock::now() - data.start_time;
-    out(cout) << "Clean up completed and total clean up time :" << timePassed.count() << std::endl;
+    out(cerr) << "Clean up completed and total clean up time :"
+              << timePassed.count() << std::endl;
 
     // lastrun file
     r = sync_remove(run_name_meta);
@@ -1244,7 +1245,8 @@ int ObjBencher::clean_up(int num_objects, int prevPid, int concurrentios) {
 
   completions_done();
 
-  out(cout) << "Removed " << data.finished << " object" << (data.finished != 1 ? "s" : "") << std::endl;
+  out(cerr) << "Removed " << data.finished << " object"
+            << (data.finished != 1 ? "s" : "") << std::endl;
 
   return 0;
 
@@ -1431,7 +1433,8 @@ int ObjBencher::clean_up_slow(const std::string& prefix, int concurrentios) {
 
   completions_done();
 
-  out(cout) << "Removed " << data.finished << " object" << (data.finished != 1 ? "s" : "") << std::endl;
+  out(cerr) << "Removed " << data.finished << " object"
+            << (data.finished != 1 ? "s" : "") << std::endl;
 
   return 0;
 

--- a/src/tools/rados/rados.cc
+++ b/src/tools/rados/rados.cc
@@ -3434,7 +3434,6 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
       object_size = op_size;
     else if (object_size < op_size)
       op_size = object_size;
-    cout << "hints = " << (int)hints << std::endl;
     ret = bencher.aio_bench(operation, seconds,
 			    concurrent_ios, op_size, object_size,
 			    max_objects, cleanup, hints, run_name, reuse_bench, no_verify);

